cdap-ui/app/directives/widget-container/widget-schema-editor/widget-schema-editor.less

```diff
@@ -17,5 +17,12 @@
 @import "../../../styles/variables.less";
 
 my-schema-editor {
-
+  .table td {
+    &.active-cell {
+      outline: 1px solid @cdap-orange;
+      &:hover {
```

Comment:
@ajainarayanan line 147 of theme.less sets outline: none for <td> and <th> on hover. 99% of the time this is what we want, but in this scenario, we want to provide a specific value for outline on hover so that it doesn't switch to none.